src/js/components/Select/Select.js

```diff
@@ -101,7 +102,10 @@ class Select extends Component {
       }
     };
 
-    const SelectIcon = theme.select.icons.down;
+    let SelectIcon;
+    if (icon !== false) {
+      SelectIcon = icon !== true ? icon : theme.select.icons.down;
```

Comment:
breaks default state of icon undefined..
maybe a switch would be easier to read?
```
let SelectIcon;
switch (icon) {
  case false:
    break;
  case true:
    SelectIcon = theme.select.icons.down;
    break;
  case undefined:
    SelectIcon = theme.select.icons.down;
    break;
  default:
    SelectIcon = icon;
}
```

Reply:
great suggestion. let me work on it.